Offseason2022/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -216,6 +216,7 @@ public void periodic( )
     updateOdometry( );
     updateDashboardValues( );
     m_field.setRobotPose(m_odometry.getPoseMeters( ));
+    SmartDashboard.putBoolean("DT_throttleZeroed", m_throttleZeroed);

```

Comment:
Can we put this line in "updateDashboardValues method, since it is updating a dashboard value? You'll see a bunch of other variables being put on the dashboard there.